### PR TITLE
projectUVfromMesh

### PR DESCRIPTION
**Pull request by Pryme8**
trying to support a simple plane UV projection method.

### CODE REVIEW

File: src/Mesh/babylon.mesh.ts [hunk 1]
```diff
@@ -1061,6 +1061,56 @@
             this._geometry.toLeftHanded();
             return this;
         }
+        
+        /**
```
Comment (deltakosh) [inline src/Mesh/babylon.mesh.ts]: Hello not sure this is for the framework. I would more see it in the snippets in the Doc (gamelets or utilities)

Comment (Pryme8) [reply src/Mesh/babylon.mesh.ts]: Aw, did I at least do my first TS contribution correctly?

Not to pressed if it gets in or not, just wanted to see if I am matching the conventions correctly or not.

Where are the gamelets or utilities, so I can look up a reference for presenting this info.

**Comment by Pryme8** [comment]
Thank you, I have been dreading learning TS but when in Rome...

**Comment by deltakosh** [comment]
Appreciated!

**Pull request closed without merging**